CONFIGURATION.md

```diff
@@ -29,3 +29,7 @@ To include additional arguments to pass to the `npm ci` command, set the environ
 ## Remove stale directories from dependency target folder
 
 To remove stale directories from the dependency target folder, set the environment variable `FALLBACK_DEPENDENCIES_REMOVE_STALE_DIRECTORIES` to `true` or set `removeStaleDirectories` in `fallbackDependencies` package.json config.
+
+## Enable checkout instead of default recloning for dependency versions
```

Comment:
we can remove this entire section from this document. the documentation under USAGE is sufficient